benchmarks/main.cpp

```diff
@@ -1,6 +1,6 @@
 //#define TEST_EASTL
 //#define TEST_FOLLY
-#define TEST_G3D_ALLOC
+//#define TEST_G3D_ALLOC
```

Comment:
If it is enabled here, we cannot disable it by compiler options.

But if it is just not defined here, we can easily enable it by compiler options e.g. `-DTEST_G3D_ALLOC`.